src/libcephfs.cc

```diff
@@ -605,18 +621,18 @@ extern "C" int ceph_rename(struct ceph_mount_info *cmount, const char *from,
 }
 
 // dirs
-extern "C" int ceph_mkdir(struct ceph_mount_info *cmount, const char *path, mode_t mode)
+extern "C" int ceph_mkdir(struct ceph_mount_info *cmount, const char *path, mode_t mode = DEFAULT_DIR_PERMISSION)
```

Comment:
I am actually struck I am unable figure out where to add the default directory and file permissions? Is this the correct location?

Reply:
No, that would be in your shell.